documentation/data/link-type.md

```diff
@@ -32,166 +32,166 @@ Each `source` and `target` object has the following keywords:
 | `property` | Optional name of a property defined in the Type definition to be used by the link relationship. |
 
 To relate instances of data specify the following options. For an instance of a Type, specify the `typeid` and optionally `index` of the instance.
-For an instance of a Container, specify the `containerid`. 
+For an instance of a Container, specify the `containerid`.
 
 ### Link Examples for Instance Data
 
 For example, to relate an instance of a Plant, specified by its indexed property PlantId with value WTP1, to an instance of Tank, specified with by its indexed property TankName with value Tank1, use the following `__Link`:
 
-    [{ 
-        "typeid": "Plant", 
-        "values": [{ 
-			"PlantId": "WTP1", 
-			"PlantName": "Water Treatment Plant One" 
-		}] 
-    }, { 
-        "typeid": "Tank", 
-        "values": [{ 
-			"TankName": "Tank1" 
-		}] 
-    }, { 
-        "typeid": "__Link", 
-        "values": [ { 
-			"source": { 
-				"typeid": "Plant", 
+    [{
+        "typeid": "Plant",
+        "values": [{
+			"PlantId": "WTP1",
+			"PlantName": "Water Treatment Plant One"
+		}]
+    }, {
+        "typeid": "Tank",
+        "values": [{
+			"TankName": "Tank1"
+		}]
+    }, {
+        "typeid": "__Link",
+        "values": [ {
+			"source": {
+				"typeid": "Plant",
 				"index": "WTP1"
-			}, 
-            "target": { 
-				"typeid": "Tank", 
-				"index": "Tank1" 
-			} 
+			},
+            "target": {
+				"typeid": "Tank",
+				"index": "Tank1"
+			}
 		}]
     }]
 
 To associate the container, Tank1Measurements, with the instance of a Tank whose index is \'Tank1\', use the following `__Link`:
 
-    [{  
-		"typeid": "__Link", 
-        "values": [{ 
-			"source": { 
-				"typeid": "Tank", 
+    [{
+		"typeid": "__Link",
```

Comment:
Examples in this file are not formatted correctly but we don't have to tackle it in this PR is more of a note.

Reply:
Oh wow I didn't notice that, I have fixed them

Reply:
Thank you, Max!